cpu/stm32_common/periph/dma.c

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_cortexm_common
```

Comment:
should be cpu_stm32_common (which means the same mistake is done in some other periph drivers ;))